lib/reporters/tap.js

```diff
@@ -62,17 +62,23 @@ class TapReporter {
 
 		const title = stripAnsi(test.title);
 
+		const appendLogs = () => {
+			if (test.logs) {
+				test.logs.forEach(log => {
+					output.push(`  ${log}`);
```

Comment:
Good catch! I'm going to make it so that it will result in something like this, where I add the `i` figure to the first line.

```
  ✔ file-one › passing test
    ℹ a comment for passing test
      with a newline
    ℹ a comment for passing test without a newline
```